pkg/fileutils/read.go

```diff
@@ -38,17 +39,21 @@ func Read(file string, base string) ([]byte, error) {
 }
 
 func readAbsFile(file string) ([]byte, error) {
+	// The file argument could be an abs file reference or a URL
 	u, err := url.Parse(file)
 	if err != nil {
 		return nil, err
 	}
-	if u.Scheme == "" {
+	if u.Scheme == "http" || u.Scheme == "https" || u.Scheme == "file" {
+		return ReadUrl(u, 0)
+	}
+	if u.Scheme == "" || onWindowsSchemeIsDriveLetter(u) {
```

Comment:
I think this disjunction should be factored out into a helper function called something like `emptyScheme` because that's really what we are looking for. That will also make the caller more self-documenting.